package.json

```diff
@@ -31,12 +31,12 @@
   },
   "homepage": "https://github.com/textlint-ja/textlint-rule-no-insert-dropping-sa",
   "devDependencies": {
-    "textlint-scripts": "^3.0.0"
+    "textlint-scripts": "^12.0.1"
   },
   "dependencies": {
     "array-find-index": "^1.0.2",
     "kuromojin": "^3.0.0",
     "morpheme-match-all": "^2.0.5",
-    "textlint-tester": "^5.3.4"
+    "textlint-tester": "^12.0.0"
```

Comment:
fmm, It is strange.
We should move it to devDeps

Reply:
`textlint-tester` was added to dependencies in https://github.com/textlint-ja/textlint-rule-no-insert-dropping-sa/pull/7.
I moved it to devDependencies.